Scribe/ChangeHistory/ChangeTestForm.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Windows.Forms;
+
+namespace Scribe.ChangeHistory
+{
+    /// <summary>
+    /// A simple form used to test the undo feature before databinding is finished.
+    /// </summary>
+    public partial class ChangeTestForm : Form
+    {
+        /// <summary>The backing field for <see cref="OldValue"/>.</summary>
+        private string _oldValue = "";
+
+        /// <summary>
+        /// Tracks whether or not the form's state has changed.
```

Comment:
nit: Perhaps this comment can be fleshed out. How does this track whether or not the form's state has changed? We'd expect a boolean value with that sort of comment.

Reply:
Although, this is as you mention test code. Going to make comments as if it wasn't test code, in case you keep some.

Reply:
This is a good point, I will expand upon it.